museek/plugin/rawdata_flagger_plugin.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+from ivory.plugin.abstract_plugin import AbstractPlugin
+from ivory.utils.requirement import Requirement
+from ivory.utils.result import Result
+from museek.enums.result_enum import ResultEnum
+from museek.factory.data_element_factory import FlagElementFactory
+from museek.flag_list import FlagList
+from museek.time_ordered_data import TimeOrderedData
+from museek.visualiser import waterfall
+
+
+class RawdataFlaggerPlugin(AbstractPlugin):
+    """ Plugin to flag raw data with values below a minimum """
+
+    def __init__(self,
+                 flag_minimum: float):
+        """
+        Initialise
+        :param flag_minimum: lower threshold to flag the data
```

Comment:
please mention in the docstring that this threshold relates to raw correlator units without any normalisation.

Reply:
mentioned